README.md

```diff
@@ -1,165 +1,261 @@
-```
-adapt update {name of plugin}
-```
-You can also update specific plugins groups:
-```
-adapt update components extensions
-```
-Run `adapt help update` for more information.
-
-##### Uninstalling a plugin from your current directory
-```
-adapt uninstall {name of plugin}
-```
-
-##### 'Developer' install
-```
-adapt devinstall
-```
-This command will clone the framework and all the plugins defined in adapt.json as git repos into a directory called 'adapt_framework'.
-
-Once the command has finished running, you will need to change to that directory and run `npm install` before you can use commands like `grunt dev`.
-
-Run `adapt help devinstall` for more information.
-
-The Plugin Registry
+[![Build Status](https://travis-ci.org/adaptlearning/adapt-cli.png?branch=master)](https://travis-ci.org/adaptlearning/adapt-cli)  [![Join the chat at https://gitter.im/adaptlearning/adapt-cli](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/adaptlearning/adapt-cli?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)    
+
+The **Adapt CLI** is a command line interface for use with the [Adapt framework](https://github.com/adaptlearning/adapt_framework). Its primary usefulness is to install, update, and uninstall Adapt plug-ins. In doing so, it references the [Adapt Plugin Browser](https://www.adaptlearning.org/index.php/plugin-browser/). Consequently, the CLI includes several commands for working with this plug-in registry. 
+
+>IMPORTANT: The **Adapt CLI** is not intended to be used with courses in the [Adapt authoring tool](https://github.com/adaptlearning/adapt_authoring). The authoring tool tracks versions of plug-ins in its database. Using the CLI bypasses this tracking.
+
+## Installation
+
+Before installing the Adapt CLI, you must install [NodeJS](http://nodejs.org) and [git](http://git-scm.com/downloads).  
+To install the Adapt CLI globally, run the following command:  
+
+`npm install -g adapt-cli`
+
+Some systems may require elevated permissions in order to install a script globally.
+
+
+## Commands  
+
+### adapt version  
+
+##### Model:  
+`adapt version` 
+
+This command reports both the version of the CLI and the version of the Adapt framework. The version of the framework is reported as "0.0.0" unless the command is run from the root directory of an installed framework. 
+<div float align=right><a href="#top">Back to Top</a></div>  
+
+### adapt help  
+
+##### Model:  
+`adapt help [<command>]` 
+
+**command**: A CLI command.  
+
+##### Examples:  
+
+1. To list all the CLI commands with a brief description of each:  
+`adapt help`  
+
+2. To report a brief description of a specific command:  
+`adapt help create`  
+ 
+<div float align=right><a href="#top">Back to Top</a></div>
+
+### adapt create  
+
+##### Model:  
+`adapt create <type> <path> [<branch>]`
```

Comment:
`<type>` and `<path>` are also optional here.

Reply:
done